tunedev/Banka
# #165057958 Add post new account route

#### What does this PR do?
adds post api/v1/accounts route to the endpoints of the server-side

#### Description of Task to be completed?
- rename some files to reduce the name length
- add test cases for a successful request and edge cases
- validate user's input

#### How should this be manually tested?
- Clone this repository
- Run npm install to get the dependencies
- Run npm start to start the app's server 
- Then make a post request to api/v1/accounts. The body should contain an existing user's id and account's type(either savings or current), a response containing new account should be returned

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[Finishes #165057958](https://www.pivotaltracker.com/story/show/165057958)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
